File: src/main/java/me/itzisonn_/meazy/runtime/EvaluationException.java
package me.itzisonn_.meazy.runtime;

import me.itzisonn_.meazy.lang.TextException;
import me.itzisonn_.meazy.lang.text.Text;
import org.jspecify.annotations.NullMarked;

/**
 * Is thrown TODO
 */
@NullMarked
public class EvaluationException extends TextException {
    /**
     * @param text Text
     */
    public EvaluationException(Text text) {
        super(text);
    }
}

File: src/main/kotlin/me/itzisonn_/meazy/runtime/EvaluationException.kt
package me.itzisonn_.meazy.runtime

import me.itzisonn_.meazy.util.text.Text
import me.itzisonn_.meazy.util.text.TextException

/**
 * Is thrown TODO
 * @param text Text
 */
class EvaluationException(text: Text) : TextException(text)